### ISSUE TITLE
[terra-alert] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3048]
# Bug Report

The following themeable variables are missing (we should theme them even though they're dev site only for example purposes):

**clinical-lowlight-theme**

--terra-alert-attention-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 9, column: 4)
--terra-alert-dismiss-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 17, column: 4)
--terra-alert-help-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 13, column: 4)

**orion-fusion-theme**

--terra-alert-attention-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 9, column: 4)
--terra-alert-dismiss-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 17, column: 4)
--terra-alert-help-example-color - packages/terra-alert/lib/terra-dev-site/doc/example/colors.module.scss (line: 13, column: 4)

**Comment by neilpfeiffer** [comment]
For the Lowlight theme:
```
:local {
  .clinical-lowlight-theme {
    /*{ ...existing styles here...}*/

    // Used for docsite custom color examples only 
    --terra-alert-attention-example-color: #f94e4e;
    --terra-alert-dismiss-example-color: #ff049b;
    --terra-alert-help-example-color: #0addff;
  }
}
```

----

For the Fusion theme:
```
:local {
  .orion-fusion-theme {
    /*{ ...existing styles here...}*/

    // Used for docsite custom color examples only 
    --terra-alert-attention-example-color: #fa0000;
    --terra-alert-dismiss-example-color: #ff049b;
    --terra-alert-help-example-color: #b961cf;
  }
}
```
